src/const.ts

```diff
@@ -17,6 +17,8 @@ export const ALLOWANCE_MAX_VALUE = TWO.pow(new BN(256)).sub(ONE) // 115792089237
 export const ALLOWANCE_FOR_ENABLED_TOKEN = TWO.pow(new BN(128)) // 340282366920938463463374607431768211456
 // How much a sell order must be selling to be considered `unlimited`
 export const MIN_UNLIMITED_SELL_ORDER = ALLOWANCE_MAX_VALUE.div(TEN).mul(new BN(7)) // Currently set to 70% of max allowance
+// Amount for an order to be considered unlimited, from contract's point of view: https://github.com/gnosis/dex-contracts/blob/master/contracts/BatchExchange.sol#L35
+export const UNLIMITED_ORDER_AMOUNT = TWO.pow(new BN(128))
```

Comment:
Out of curiousity, does this not need ot be `TWO.pow(new BN(128)).sub(ONE)`? or overflow errors happen?

Reply:
Good catch, even excellent catch :+1: 
`uint128(-1)` in the contract is converted into max value for uint128 (or at least I expect it to work like that in solidity ), which is `340282366920938463463374607431768211455`, or in JS `2n ** 128n  - 1`

Reply:
So I need to subtract 1 from this amount?